packages/builder/src/components/start/Steps/API.svelte

```diff
@@ -11,7 +11,7 @@
     on:input={() => (blurred.api = true)}
     label="API Key"
     name="apiKey"
-    placeholder="Enter your API Key"
+    placeholder="Use command-V to paste your API Key"
```

Comment:
Most keyboards don't have a command key. We probably need to just be more obvious that the hosting portal is copying to clipboard.

Reply:
The real issue is that in electron we dont have a "Right Click" > "Paste" menu... there is no reight click menu at all.

I briefly looked into adding a "Paste From Clipboard" button - but needed to use `const { clipboard } = require("electron")`  on the server, then have an http endpoint to pass the result to the client!

In the end i opted for this.

Going to leave it... the point is just to make people aware to use Ctrl+v / Cmd+V  - windows users will get it i think.